tests/foreman/longrun/test_inc_updates.py

```diff
@@ -0,0 +1,305 @@
+            content_view=cls.rhel_6_partial_cv,
+            type='erratum',
+            name='rhel_6_partial_cv_filter',
+            repository=[cls.rhel_66_repo]
+        ).create()
+
+        # Step 9: Create a content view filter rule - filtering out errata in
+        # the last 365 days
+        start_date = (date.today() - timedelta(days=365)).strftime('%Y-%m-%d')
+        ContentViewFilterRule(
+            content_view_filter=rhel_6_partial_cvf,
+            types=['security', 'enhancement', 'bugfix'],
+            start_date=start_date,
+            end_date=date.today().strftime('%Y-%m-%d')
+        ).create()
+
+        # Step 10: Publish both the content views and re-read the content views
+
+        # Changing the nailgun timeout time for the rest of the steps as
+        # publish/promote of larger repos take more than 5 minutes
+        entity_mixins.TASK_TIMEOUT = 3600
+        for content_view in (cls.rhel_6_cv, cls.rhel_6_partial_cv):
+            content_view.publish()
+        cls.rhel_6_cv = cls.rhel_6_cv.read()
+        cls.rhel_6_partial_cv = cls.rhel_6_partial_cv.read()
+
+        # Step 11: Promote both content views to 'DEV' and 'QE'
+        for content_view in (cls.rhel_6_cv, cls.rhel_6_partial_cv):
+            assert len(content_view.version) == 1
+            for env in (cls.dev_lce, cls.qe_lce):
+                promote(content_view.version[0], env.id)
+
+        # Step 12: Create host collections
+        for hc_name in ('rhel_6_hc', 'rhel_6_partial_hc'):
+            setattr(
+                cls,
+                hc_name,
+                HostCollection(
+                    organization=cls.org, name=hc_name, max_content_hosts=5
+                ).create()
+            )
+
+        # Step 13: Create activation keys for both content views
+        kwargs = {'organization': cls.org, 'environment': cls.qe_lce.id}
+        rhel_6_ak = ActivationKey(
+            name=u'rhel_6_ak',
+            content_view=cls.rhel_6_cv,
+            host_collection=[cls.rhel_6_hc],
+            **kwargs
+        ).create()
+        rhel_6_partial_ak = ActivationKey(
+            name=u'rhel_6_partial_ak',
+            content_view=cls.rhel_6_partial_cv,
+            host_collection=[cls.rhel_6_partial_hc],
+            **kwargs
+        ).create()
+
+        # Step 14: Assign subscriptions to activation keys
+        # Fetch available subscriptions
+        subs = Subscription(organization=cls.org).search()
+        assert len(subs) > 0
+
+        # Add subscriptions to activation key
+        print 'Adding subscriptions to both Activation keys'
+        sub_found = False
+        for sub in subs:
+            if sub.read_json()['product_name'] == u'Employee SKU':
+                for act_key in [rhel_6_ak, rhel_6_partial_ak]:
+                    act_key.add_subscriptions(data={
+                        u'subscription_id': sub.id
+                    })
+                sub_found = True
```

Comment:
should we add a `break` here to avoid looping through all other possible iterations? Not a blocker